packages/notification-center/src/components/notification-center/components/user-preference/styles.ts

```diff
@@ -71,8 +73,7 @@ export const switchStyles = (baseTheme: IThemeUserPreferences) => {
   };
 };
 
-export const Text = styled.div<{ color: string; size: 'sm' | 'md' | 'lg' }>`
-  color: ${({ color }) => color};
+export const Text = styled.div<{ size: 'sm' | 'md' | 'lg' }>`
```

Comment:
I want to do a type for size and share it in the monorepo.

Reply:
yeah, will be useful I guess, but I don't see where we do define these, shared I guess?

Reply:
Yeah. We will do that in another piece of work. We have to improve the typings.